maro/rl/policy_v2/policy_base.py

```diff
@@ -395,15 +395,20 @@ def __init__(self, name: str, device: str) -> None:
         """
         super(MultiRLPolicy, self).__init__(name=name, device=device)
 
-    def _shape_check(self, states: np.ndarray, actions: Union[None, List[np.ndarray]]) -> bool:
-        if not (states.shape[0] > 0 and match_shape(states, (None, self.state_dim))):
-            return False
+    def __call__(self, states: List[np.ndarray], agent_ids: List[int]):
+        assert self._shape_check(states=states, actions=None)
+        ret = self._call_impl(states, agent_ids)
```

Comment:
`AbsRLPolicy._call_impl()` does not have parameter `agent_ids`. Personally, I think we need a new abstraction for multi RL policies, for example: `AbsMultiPolicy => AbsMultiRLPolicy => ...`